### PR TITLE
[MICROBA-744] Added the edx.org theme to support future edx theme

### PR DESCRIPTION
**Pull request by staubina**
efforts.

The changes included will allow us to change the theme of the Learner
Record and keep the edx specific brand changes in this repo.

### CODE REVIEW

**Comment by staubina** [comment]
All of the CSS files here are copied from Credentials. I added the _base_style.html file and changed the webpack config.

File: webpack.config.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const webpack = require('webpack');
```
Comment (staubina) [inline webpack.config.js:4]: Note: File changed.

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
   "homepage": "https://github.com/edx/credentials-themes",
   "devDependencies": {
     "@edx/edx-bootstrap": "^1.0.3",
+    "@edx/paragon": "^12.2.0",
```
Comment (staubina) [inline package.json:18]: Note: Installed Paragon to compile

File: edx_credentials_themes/templates/edx.org/_base_style.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
```
Comment (staubina) [inline edx_credentials_themes/templates/edx.org/_base_style.html:1]: Note: New HTML file.

**Review by Tj-Tracy — APPROVED** [review]

**Pull request merged**